dmc/customers/services/github_service/octokit_github_service.rb

```diff
@@ -38,6 +38,13 @@ def reset_tokens(oauth_token)
       client.reset_application_authorization(oauth_token)
     end
 
+    def create_webhook(oauth_token, full_name)
+      client(oauth_token).create_hook(full_name, 'web', {
+        url: 'http://api.deploymy.codes/webhook/github',
```

Comment:
Could you move this URL in a configuration file? Because it's not going to be the same for all the environments. At least, the scheme and host. The path can stay here.